traffic_ops/traffic_ops_golang/servercheck/servercheck.go

```diff
@@ -131,6 +131,16 @@ func CreateUpdateServercheck(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	cdnName, err := dbhelpers.GetCDNNameFromServerID(inf.Tx.Tx, int64(id))
```

Comment:
`servercheck` probably doesn't need to be lock-safe since it's primarily just for monitoring and doesn't really affect CDN operation